mulog-kinesis/test/com/brunobonacci/mulog/publishers/test_publisher.clj

```diff
@@ -0,0 +1,56 @@
+(ns com.brunobonacci.mulog.publishers.test-publisher
+  (:require [com.brunobonacci.mulog :as μ]
+            [cheshire.core :as json]
+            [cognitect.aws.client.api :as aws])
+  (:import (java.util.concurrent TimeUnit)))
+
+(def KINESIS-LOCAL-STREAM-NAME "mulog-test-stream")
+
+(def KINESIS-LOCAL-SETTINGS {:api :kinesis
+                             :endpoint-override
+                                  {:protocol :http
+                                   :hostname "localhost"
+                                   :port 4568}})
+
+(defmacro with-local-kinesis-client
+  []
+  `(aws/client KINESIS-LOCAL-SETTINGS))
```

Comment:
there is no need for this to be a macro, I can be just a normal function

`with-` prefix is typically used to wrap code
I would call this function `kinesis-local-client` or similar